added some info that was missing (#2560)
umbraco/UmbracoDocs: Umbraco-Cloud/Troubleshooting/Deployments/Deploy-Settings/index.md

  <deploy sessionTimeout="1200" sourceDeployTimeout="1200" httpClientTimeout="1200"/>
</settings>
```

> Do note that those settings have to be added on both the source and target environments to work.

Reviewer: What do you think of adding this as a **note** instead? 🤔 

```suggestion
:::note
It's important that these settings are added to both the source and target environments in order to work.
:::
```

Author: Sure, the note is even better 😊
